vmc/resource_vmc_cluster.go

```diff
@@ -24,7 +24,15 @@ func resourceCluster() *schema.Resource {
 		Update: resourceClusterUpdate,
 		Read:   resourceClusterRead,
 		Importer: &schema.ResourceImporter{
-			State: schema.ImportStatePassthrough,
+			State: func(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+				idParts := strings.Split(d.Id(), ",")
+				if len(idParts) != 2 || idParts[0] == "" || idParts[1] == "" {
+					return nil, fmt.Errorf("Unexpected format of ID (%q), expected sddc_id,cluster_id", d.Id())
```

Comment:
space after sddc_id,

Reply:
No space is needed since that is the format the user should specify in order to import the resource. For example, terraform import vmc_cluster.cluster_1 afe7a0fd-3f0a-48b2-9ddb-0489c22732ae,45495963-d24d-469b-830a-9003bfe132b5